examples/chart/teleport-cluster/templates/auth/_config.azure.tpl

```diff
@@ -0,0 +1,38 @@
+{{/* Helper to build the database connection string, adds paraneters if needed */}}
+{{- define "teleport-cluster.auth.config.azure.conn_string.query" }}
+  {{- if .Values.azure.databasePoolMaxConnections -}}
+    {{- printf "sslmode=verify-full&pool_max_conns=%v" .Values.azure.databasePoolMaxConnections -}}
+  {{- else -}}
+    sslmode=verify-full
+  {{- end -}}
+{{- end -}}
+
+{{- define "teleport-cluster.auth.config.azure" -}}
+{{ include "teleport-cluster.auth.config.common" . }}
+  storage:
+    type: postgresql
+    auth_mode: azure
+    conn_string: {{ urlJoin (dict
+      "scheme" "postgresql"
+      "userinfo" .Values.azure.databaseUser
+      "host" (printf "%v.postgres.database.azure.com" .Values.azure.databaseHost)
```

Comment:
can we add a check to prevent users from setting the postgres.database.azure.com?